front/app/containers/Admin/projectFolders/containers/permissions/index.tsx

```diff
@@ -222,34 +217,30 @@ const FolderPermissions = ({
         <List>
           <>
             {!isNilOrError(folderModerators) &&
-              !isNilOrError(authUser) &&
               folderModerators.map((folderModerator, index) => (
+                // This row is a near copy of ModeratorListRow. With some moderations
```

Comment:
I think you meant `With some modifications` here. Also this is a comment that is most likely had to maintain. `ModeratorListRow` can be changed and this comment would wrong 🤔 . Happy to leave as is but but maybe we could leave a tech debt ticket to follow it up and re-use `ModeratorListRow`

Reply:
Changed the comment. Not yet logging a ticket. It's only duplicated in 1 spot now, so not high priority.